packages/apsara-ui/src/Header/Header.tsx

```diff
@@ -1,30 +1,28 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-nocheck
 
-import React, { ReactChildren } from "react";
-import { Layout, Avatar, Menu, Dropdown } from "antd";
+import React, { ReactChildren, useContext } from "react";
+import { ThemeContext } from "styled-components";
+import { Dropdown } from "antd";
 import Lottie from "react-lottie";
 import CustomIcon from "../Icon";
 import Learn from "../Learn";
 import Markdown from "../Markdown";
 import learnAnimation from "./learn.json";
+import { Wrapper, RightBar, Title, Holder, Avatar, Menu, MenuLink } from "./Header.styles";
 
-import "./header.less";
-const AntHeader = Layout.Header;
-
-const getMenuList = (menuList: any[]) => (
-    <Menu className="header--wrapper--dropdown">
+const getMenuList = (menuList: any[], iconColor: string) => (
+    <Menu>
         {menuList.map((menu: any) => (
             <Menu.Item key={menu.name}>
-                <a
+                <MenuLink
                     rel="noopener noreferrer"
                     href={menu.href ? menu.href : "#"}
                     onClick={menu.onClick ? menu.onClick : () => null}
-                    className="header--wrapper--dropdown-item"
                 >
-                    {menu.icon && <CustomIcon name={menu.icon} size={24} styleOverride={{ color: "#4C4C4C" }} />}
+                    {menu.icon && <CustomIcon name={menu.icon} size={24} styleOverride={{ color: iconColor }} />}
```

Comment:
It looks like these menu icons are left-aligned with the logo Icon. Let's have a 32 width parent container with an align-item center. wdyt?